tests/test_prettytable.py

```diff
@@ -227,7 +229,16 @@ def testSetForAllColumns(self):
         self.x.vertical_char = "!"
         self.x.horizontal_char = "~"
         self.x.junction_char = "*"
+        self.x.top_junction_char = "@"
+        self.x.bottom_junction_char = "#"
+        self.x.right_junction_char = "$"
+        self.x.left_junction_char = "%"
+        self.x.top_right_junction_char = "^"
+        self.x.top_left_junction_char = "&"
+        self.x.bottom_right_junction_char = "("
+        self.x.bottom_left_junction_char = ")"
         self.x.format = True
+        self.x.xhtml = True
```

Comment:
Is this needed?

Reply:
It is needed only for coverage for the property setter.
Do you prefer to omit this change?

Reply:
And let's remove this too.

Reply:
Removed